spec/features/feedex_spec.rb

```diff
@@ -44,7 +42,28 @@
         "URL" => "/vat-rates",
         "Referrer" => "https://www.gov.uk/pay-vat"
       }
-    ])
+    ]
+
+    explore_anonymous_feedback_with(url: "https://www.gov.uk/vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
+
+    explore_anonymous_feedback_with(url: "http:/www.gov.uk/vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
+
+    explore_anonymous_feedback_with(url: "https//www.gov.uk/vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
+
+    explore_anonymous_feedback_with(url: "www.gov.uk/vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
+
+    explore_anonymous_feedback_with(url: "gov.uk/vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
+
+    explore_anonymous_feedback_with(url: "/vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
+
+    explore_anonymous_feedback_with(url: "vat-rates")
+    expect(feedex_results).to eq(feedback_reports)
```

Comment:
It'd be best to test this logic in a unit test, because here:

- it makes the integration test harder to read
- it distracts from the main use-case
- it makes the integration tests slower (because they run through the UI)